flow/src/main/scala/com/tersesystems/echopraxia/plusscala/flow/DefaultFlowLoggerMethods.scala

```diff
@@ -97,14 +97,14 @@ trait DefaultFlowLoggerMethods[FB <: FlowFieldBuilder] extends DefaultMethodsSup
 
   @inline
   private def execute[B: ToValue](level: JLevel, attempt: => B): B = {
-    core.log(level, fieldBuilder.enteringTemplate, entering, fieldBuilder)
-
+    val handle: LoggerHandle[FB] = core.logHandle(level, fieldBuilder);
+    handle.log(fieldBuilder.enteringTemplate, entering)
```

Comment:
Calling `core.log` repeatedly re-evaluates conditions that should only happen once -- we need a handle here directly to the logger, and we need the check to handle outside of this method.